src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -138,6 +141,7 @@ describe('<CourseRunEnrollment />', () => {
 
     screen.getByRole('button', { name: 'Enroll now' });
     screen.getByText('Your enrollment request failed.');
+    expect(spyConsole).toHaveBeenCalled();
```

Comment:
To go further, we could use `toHaveBeenCalledWith` to also test the value received by `handle`. In this way our tests will failed if error code change.